_includes/2017_data/sponsors.html

```diff
@@ -12,6 +12,14 @@ <h2 class="section-heading" id="sponsors">Sponsors</h2>
                     <h3 class="section-heading">Metropolis-Hastings Tier</h3>
                 </div>
             </div>
+
+            <div class = "row">
+	    	<div class = "col-md-4 col-sm-6">
+			<a href = "https://zulip.com/" class = "portfolio-link" data-toggle = "modal">
+			<img src = "./css/2017_style/img/sponsors/zulip.png" class = "image-responsive img-centered" alt = "zulip">
+			</a>
+		</div>
+	    </div>
```

Comment:
Not sure if you are able to preview the website locally, but this looks like it's adding the zulip logo beside the tidelift one, in the "metropolis hastings" tier

What should happen is adding a new section with a header like the metropolis hasting tier one below it, the new header should say "Technology partners" and have the zulip logo under this new header

Reply:
Oh ok...understood, will change that also